drivers/edac/shell.c

```diff
@@ -0,0 +1,389 @@
+/*
```

Comment:
You are missing commit body for "edac: shell: .." commit.

Reply:
fixed